doc/source/whatsnew/v0.13.1.rst

```diff
@@ -222,60 +222,6 @@ Enhancements
 
 - Panel :meth:`~pandas.Panel.apply` will work on non-ufuncs. See :ref:`the docs<basics.apply>`.
 
-  .. ipython:: python
-
-     import pandas.util.testing as tm
```

Comment:
Hmm I don't think we want to change old whatsnew items like this but I'm not sure how we've handled removals like this historically either - @TomAugspurger maybe you can offer insight?

Reply:
yeah don't change these at all, rather convert them to code-blocks